### PR TITLE
[zh-cn] init translation of 'Test your skills: Backgrounds and borders'

### LINKED ISSUE #12949: zh-CN: Mislocated tests on `Backgrounds_and_borders`
### MDN URL

https://developer.mozilla.org/zh-CN/docs/Learn/CSS/Building_blocks/Backgrounds_and_borders

### What specific section or headline is this issue about?

玩转背景和边框

### What information was incorrect, unhelpful, or incomplete?

[英文版](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Backgrounds_and_borders#test_your_skills!)已将此测验移动至[单独页面](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Test_your_skills_backgrounds_and_borders#task_1)。

### What did you expect to see?

- [x] 更新 https://developer.mozilla.org/zh-CN/docs/Learn/CSS/Building_blocks/Backgrounds_and_borders 中的此节；#13474
- [ ] 翻译 https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Test_your_skills_backgrounds_and_borders 。#13498

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

building blocks 下的其他页面可能也有此问题，但是因为这不是我目前的主线任务。所以我没有排查其他页面。

关联议题：https://github.com/mdn/css-examples/issues/36

### PR DESCRIPTION
**Pull request by jasonren0403**
### Related issues and pull requests

closes #12949

### CODE REVIEW

File: files/zh-cn/learn/css/building_blocks/test_your_skills_backgrounds_and_borders/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: 技能测试：背景与边框
+slug: Learn/CSS/Building_blocks/Test_your_skills_backgrounds_and_borders
+---
+
+{{learnsidebar}}
+
+本技能测试的目的是评估你是否完全理解了 [CSS 中的背景和边框](/zh-CN/docs/Learn/CSS/Building_blocks/Backgrounds_and_borders)教程的内容。
+
+> **备注：** 你可以在下面的互动编辑器中尝试解决方案。然而，下载代码并使用在线工具，如 [CodePen](https://codepen.io/)、[jsFiddle](https://jsfiddle.net/) 或 [Glitch](https://glitch.com/)来完成任务可能会有所帮助。
```
Comment (skyclouds2001) [inline files/zh-cn/learn/css/building_blocks/test_your_skills_backgrounds_and_borders/index.md:10]:
```suggestion
> **备注：** 你可以在下面的互动编辑器中尝试解决方案。然而，下载代码并使用在线工具，如 [CodePen](https://codepen.io/)、[jsFiddle](https://jsfiddle.net/) 或 [Glitch](https://glitch.com/) 来完成任务可能会有所帮助。
```

File: files/zh-cn/learn/css/building_blocks/test_your_skills_backgrounds_and_borders/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+slug: Learn/CSS/Building_blocks/Test_your_skills_backgrounds_and_borders
+---
+
+{{learnsidebar}}
+
+本技能测试的目的是评估你是否完全理解了 [CSS 中的背景和边框](/zh-CN/docs/Learn/CSS/Building_blocks/Backgrounds_and_borders)教程的内容。
+
+> **备注：** 你可以在下面的互动编辑器中尝试解决方案。然而，下载代码并使用在线工具，如 [CodePen](https://codepen.io/)、[jsFiddle](https://jsfiddle.net/) 或 [Glitch](https://glitch.com/) 来完成任务可能会有所帮助。
+>
+> 如果你被卡住了，那就向我们寻求帮助吧——见本页面底部的[评估或进一步帮助](#评估或进一步帮助)部分。
```
Comment (yin1999) [inline files/zh-cn/learn/css/building_blocks/test_your_skills_backgrounds_and_borders/index.md:12]:
```suggestion
> 如果你遇到了困难，那就向我们寻求帮助吧——见本页面底部的[评估或进一步帮助](#评估或进一步帮助)部分。
```

File: files/zh-cn/learn/css/building_blocks/test_your_skills_backgrounds_and_borders/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+![图片显示了一个有照片背景的盒子，圆形的边框和半透明的黑色背景上的白色文字](backgrounds-task1.png)
+
+尝试更新下面的实时代码，重现完成的示例：
+
+{{EmbedGHLiveSample("css-examples/learn/tasks/backgrounds/backgrounds1.html", '100%', 700)}}
+
+> **标注：**
+>
+> [下载这项任务的起点](https://github.com/mdn/css-examples/blob/main/learn/tasks/backgrounds/backgrounds1-download.html)，在你自己的编辑器或在线编辑器中完成测试。
```
Comment (yin1999) [inline files/zh-cn/learn/css/building_blocks/test_your_skills_backgrounds_and_borders/index.md:32]:
```suggestion
> [下载这项任务的初始代码](https://github.com/mdn/css-examples/blob/main/learn/tasks/backgrounds/backgrounds1-download.html)，在你自己的编辑器或在线编辑器中完成测试。
```

File: files/zh-cn/learn/css/building_blocks/test_your_skills_backgrounds_and_borders/index.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+![图片显示了一个在左上角和右下角有圆角蓝色边框的盒子。文字的左边有一颗星星，右边有 3 颗星星](backgrounds-task2.png)
+
+尝试更新下面的实时代码，重现完成的示例：
+
+{{EmbedGHLiveSample("css-examples/learn/tasks/backgrounds/backgrounds2.html", '100%', 700)}}
+
+> **标注：**
+>
+> [下载这项任务的起点](https://github.com/mdn/css-examples/blob/main/learn/tasks/backgrounds/backgrounds2-download.html)，在你自己的编辑器或在线编辑器中完成测试。
```
Comment (yin1999) [inline files/zh-cn/learn/css/building_blocks/test_your_skills_backgrounds_and_borders/index.md:52]:
```suggestion
> [下载这项任务的初始代码](https://github.com/mdn/css-examples/blob/main/learn/tasks/backgrounds/backgrounds2-download.html)，在你自己的编辑器或在线编辑器中完成测试。
```

**Review by yin1999 — APPROVED** [review]
LGTM, just some suggestions. Afterwords, feel free to merge this /:

**Pull request merged**